packages/medusa/src/models/money-amount.ts

```diff
@@ -43,17 +47,27 @@ export class MoneyAmount extends SoftDeletableEntity {
   @JoinColumn({ name: "price_list_id" })
   price_list: PriceList | null
 
-  @Index('idx_money_amount_variant_id')
-  @Column({ nullable: true })
-  variant_id: string
-
-  @ManyToOne(() => ProductVariant, (variant) => variant.prices, {
+  @ManyToMany(() => ProductVariant, {
     onDelete: "CASCADE",
   })
-  @JoinColumn({ name: "variant_id" })
+  @JoinTable({
```

Comment:
question: With this pr, is there anything to think about changing in the workflows?